grails-app/views/admin/_permissionTablePaginated.gsp

```diff
@@ -0,0 +1,162 @@
+                    } else {
+                        return '<a class="btn btn-small tooltips href="" title="remove this user and role combination"><i class="icon-remove"></i></a>';
+                    }
+                },
+                bSortable : false
+            }]
+        });
+
+        $('#member-list').on( "change", "tbody td:nth-child(3) select", function (e) {
+            e.preventDefault();
+
+            var role = $(this).val();
+            var row = this.parentElement.parentElement;
+            var data = table.row(row).data();
+            var currentRole = data.role;
+            var userId = data.userId;
+
+            var message;
+            if (userId == ${user?.userId}) {
+                message = "<span class='label label-important'>Important</span><p><b>If you modify your access level you may need assistance to get it back.</b></p><p>Are you sure you want to change your access to this project from " + currentRole + " to " + decodeCamelCase(role)+"?</p>";
+            }
+            else {
+                message = "Are you sure you want to change this user's access from " + decodeCamelCase(currentRole) + " to " + decodeCamelCase(role) + "?";
+            }
+
+            bootbox.confirm(message, function(result) {
+                if (result) {
+                    addUserWithRole(userId, role, "${project.projectId}");
+
+                } else {
+                    reloadMembers(); // reload table
+                }
+            });
+        });
+
+        $('#member-list').on( "click", "tbody td:nth-child(4) a", function (e) {
+            e.preventDefault();
+
+            var row = this.parentElement.parentElement;
+            var data = table.row(row).data();
+            var userId = data.userId;
+            var role = data.role;
+
+            var message;
+            if (userId == ${user?.userId}) {
+                message = "<span class='label label-important'>Important</span><p><b>If you proceed you may need assistance to get your access back.</b></p><p>Are you sure you want to remove your access to this project?</p>";
+            }
+            else {
+                message = "Are you sure you want to remove this user's access?";
+            }
+            bootbox.confirm(message, function(result) {
+                if (result) {
+                    if (userId && role) {
+                        removeUserRole(userId, role);
+                    } else {
+                        alert("Error: required params not provided: userId & role");
+                    }
+                }
+            });
+        });
+
+        function updateStatusMessage2(msg) {
+            $('#formStatus span').text(''); // clear previous message
+            $('#formStatus span').text(msg).parent().fadeIn();
+        }
+
+        function removeUserRole(userId, role) {
+            $.ajax( {
+                url: "${createLink(controller:'user', action:'removeUserWithRoleFromProject')}",
```

Comment:
For consistency, can we move url's to top of the file.
Example: fcConfig.removeUserWithRoleFromProjectUrl